### ISSUE TITLE
Replace Taxi & Rideshare TV & Top categories with "Cars" or "Automobiles"

### ISSUE THREAD

**Issue by jayshao** [openooh/venue-taxonomy#100]
Currently we have:

Taxi & Rideshare TV | Advertising displays placed inside taxis and rideshare vehicles visible to passengers in the back seat. | 103 | transit.taxi_rideshare_tv
-- | -- | -- | --
Taxi & Rideshare Top | Advertising displays placed on top of taxi and rideshare vehicles visible to nearby pedestrian and drivers. | 104 | transit.taxi_rideshare_top

It would be nice to normalize with our other categories to have Taxi/Rideshare have a master-category - and to be able to have "inside" "outside" and maybe "platform" (if we think taxi-queues or other loading areas are an advertising area) in line with the rest of transit

**Comment by jayshao** [comment]
If we remove the strings, this seems like it's just a rename (e.g. to "Cars"). Anyone on the demand side differentiate Taxis especially?

**Comment by Tatianaabor** [comment]
I agree. We should use a more generic name so other types of vehicles can be included within this category.

**Comment by jbergmediamath** [comment]
Is there much advertising on cars outside of rideshares and taxis? May just be my ignorance.

**Comment by jayshao** [comment]
include in 1.2
